redhat-developer/developers.redhat.com
# [MWES-3156] Update Akamai to CCUV3

Akamai are ending support for v2 of their FastPurge API which is what we are currently using. Therefore we need to bump it to use v3. The Akamai Plugin supports v3 out-of-the-box, but I have taken the chance here to update the plugin to the latest version and then ensured that we're using v3 by default.

**_Please do not merge this without co-ordinating with me as I need to update firewall rules and Akamai secrets in our production environment first._**

### JIRA Issue Link
* https://issues.jboss.org/browse/MWES-3156

### Verification Process

* The build should go green
* Check that Akamai is reported at version `8.x-3.0-alpha4` on `admin/reports/updates` in Drupal
* View admin/config/akamai/config in Drupal and ensure that CCU Version is Akamai Client CCUv3
